build/gulpfile.extensions.js

```diff
@@ -189,6 +190,17 @@ gulp.task(task.define('extensions-ci', task.series(compileExtensionsBuildTask)))
 
 exports.compileExtensionsBuildTask = compileExtensionsBuildTask;
 
+// {{SQL CARBON EDIT}} Need to handle localization unlike above.
+const compileLocalizationExtensionsBuildTask = task.define('compile-localization-extensions-build', task.series(
+	task.define('clean-localization-build', util.rimraf('.locbuild')),
+	task.define('bundle-marketplace-extensions-build', () => ext.packageMarketplaceExtensionsStream(false).pipe(gulp.dest('.locbuild'))),
```

Comment:
Originally this was to be bundled with the task for updating the langpacks directly, and the id for the extensions folder was "vscode-extensions" instead of "extensions" when it appeared in the output folder. Since that step is to be done at a later time and there are ways to handle the different name. I will use the build folder directly, but note that it is not to be used for compilation, but rather for generating XLFs exclusively.